src/main/java/dev/codedsakura/blossom/lib/permissions/Permissions.java

```diff
@@ -62,4 +62,14 @@ public static boolean check(@NotNull ServerCommandSource source, @NotNull String
         }
         return source.hasPermissionLevel(level);
     }
+
+    /**
+     * @see me.lucko.fabric.api.permissions.v0.Permissions#check(Entity, String, boolean)
+     */
+    public static boolean check(@NotNull Entity entity, @NotNull String permission, boolean fallback) {
+        if (isFPAPILoaded()) {
+            return PermissionsExecutor.check(entity, permission, fallback);
+        }
+        return fallback;
+    }
```

Comment:
i think a `check(Entity, String, int)` would be useful aswell, same in the `PermissionsExecutor` file

Reply:
not sure why i didn't do that already 🤔